libs/RobotKit/tests/RobotController_test_stateIdle.cpp

```diff
@@ -161,7 +176,7 @@ TEST_F(RobotControllerTest, stateIdleDiceRollDetectedDelayOverEventAutonomousAct
 {
 	rc.state_machine.set_current_states(lksm::state::idle);
 
-	auto minimal_delay_over = 1001ms;
+	auto minimal_delay_over = 1001s;
```

Comment:
Why this change from `ms` to `s` ?

If you want a delay over 11s, it is just `11s` or `11001ms`